sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -16,31 +16,33 @@ the latest version improves on several areas from V2:
 
 ### Handling backpressure
 
-Prior to V5, events were delivered as they were received with no ability 
+Prior to V5, events were delivered as they were received with no ability
 for the user to control the pace. This could result in flooding of downstream
-dependencies as well as confusion about which events had been consumed in 
+dependencies as well as confusion about which events had been consumed in
 what order, making checkpointing difficult to do correctly.
 
-In V5 the model has been simplified so new events are not delivered until the 
-previous batch has been consumed by your event handler. You can see a sample 
+In V5 the model has been simplified so new events are not delivered until the
+previous batch has been consumed by your event handler. You can see a sample
 demonstrating this [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/typescript/src/receiveEvents.ts)
 
 ### Specific clients for sending vs receiving
 
-In V5 we've simplified the API surface, making two distinct clients, rather 
+In V5 we've simplified the API surface, making two distinct clients, rather
 than having a single `EventHubClient`:
-* [EventHubProducerClient](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubproducerclient)
+
+- [EventHubProducerClient](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubproducerclient)
   for sending messages.
-* [EventHubConsumerClient](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubconsumerclient) 
+- [EventHubConsumerClient](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubconsumerclient)
   for receiving messages.
 
-We've also merged the functionality from `EventProcessorHost` from the `@azure/event-processor-host` package into 
+We've also merged the functionality from `EventProcessorHost` from the `@azure/event-processor-host` package into
 `EventHubConsumerClient` in the `@azure/event-hubs` package, allowing `EventHubConsumerClient` to be the single
 point of entry for receiving of any type (from single partition, all partitions, or with load balancing and checkpointing features) within Event Hubs.
 
 Important note on checkpoints: `EventHubConsumerClient` does not support legacy checkpoint data.
```

Comment:
Also mention which library to use for legacy checkpoints?

Reply:
`@azure/event-process-host` is mentioned on the next line. Is that what you mean?

Reply:
Well, yes.